app/src/actions/notifyActions.js

```diff
@@ -0,0 +1,12 @@
+import * as types from './actionTypes';
+import { openSnackbar } from '../notifier/Notifier';
+
+export function notify(str) {
```

Comment:
this would work... but I think there might be an even more straight-forward way to handle this

the action would simply be the `showNotify` as you have before.

in the locations where you're currently calling `notify` (queryActions and httpService), you'd use the raw action

then in the notify component you'd wrap it with the store provider, if necessary, and then have the state of data in the `store` drive a call to open the snackbar (there in Notifier.js) whenever `this.store.notify.msg` changes or has a non-empty status

if that doesn't make sense, let's talk about it real quick

essentially, I don't think that `notifyActions` should need to depend on `Notifier` and `Notifier` shouldn't need the `openSnackbar` method to be called from outside of that component